src/pages/costModels/createCostModelWizard/steps.tsx

```diff
@@ -58,6 +58,28 @@ export const stepsHash = (t: (text: string) => string) => ({
       component: <Review />,
     },
   ],
+  GCP: [
+    {
+      id: 1,
+      name: t('cost_models_wizard.steps.general_info'),
+      component: <GeneralInformation />,
+    },
+    {
+      id: 2,
+      name: t('cost_models_wizard.steps.markup'),
+      component: <Markup />,
+    },
+    {
+      id: 3,
+      name: t('cost_models_wizard.steps.sources'),
+      component: <Sources />,
+    },
+    {
+      id: 4,
+      name: t('cost_models_wizard.steps.review'),
+      component: <Review />,
+    },
+  ],
```

Comment:
You can omit these changes to cost models.

Reply:
Done